crypto/evp/pkey_mac.c

```diff
@@ -99,6 +105,12 @@ static int pkey_mac_copy(EVP_PKEY_CTX *dst, const EVP_PKEY_CTX *src)
     EVP_PKEY_CTX_set_data(dst, dctx);
     dst->keygen_info_count = 0;
 
+    if (sctx->ctx == NULL) {
```

Comment:
This is too late to check this..
There is a if (sctx->ctx->data == NULL) check further up which is bad and will get picked up in static analysis.

Reply:
Fixed